### PR TITLE
Save nodeId as a mongo object ID in Mongo 

### PR DESCRIPTION
**Pull request by keedya**
Fix issue:
issue:https://github.com/RackHD/RackHD/issues/291
Related issues:
- Validate ssh task failed to lookup node  IP address

### CODE REVIEW

**Comment by keedya** [comment]
@benbp @zyoung51 @tannoa2 @RackHD/corecommitters @brianparry @jlongever

**Comment by benbp** [comment]
👍

**Comment by keedya** [comment]
Test this please

File: lib/models/lookup.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ function LookupModelFactory (waterline, Model, assert, Errors, Promise, Constant
 
             switch(dbType) {
                 case 'mongo':
-                    return self.findAndModifyMongo(query, {}, { $set: { node: node }}, options);
+                    return self.findAndModifyMongo(query, {}, { $set: { node: waterline.workitems.mongo.objectId(node)}}, options);
```
Comment (zyoung51) [inline lib/models/lookup.js]: Moght be worth changing this to a waterline.lookups object instead of waterline.workitems.

**Comment by zyoung51** [comment]
+1, Jenkins test this please

**Pull request merged**